receiver/zookeeperreceiver/metadata.yaml

```diff
@@ -5,25 +5,34 @@ attributes:
     description: State of the Zookeeper server (leader, standalone or follower).
   zk.version:
     description: Zookeeper version of the instance.
+  follower.state:
+    description: State of followers
+    enum:
+      - synced
+      - not_synced
```

Comment:
nit: would the word `unsynced` be appropriate?

Reply:
I'm not sure honestly this is kind of splitting hairs but to me `unsynced` seems more like it fell out of sync and `not_synced` more that it has yet to sync. I'm not partial to either one so I'll change it. Mostly users likely won't think that hard about it.